ui/org.eclipse.pde.core/src/org/eclipse/pde/core/target/LoadTargetDefinitionJob.java

```diff
@@ -185,4 +185,11 @@ private void resetPlatform(IProgressMonitor monitor) {
 		PDECore.getDefault().getFeatureModelManager().targetReloaded();
 	}
 
+	/**
+	 * @since 3.17
+	 */
+	public static Object getFamily() {
```

Comment:
Instead of adding this new API I would prefer to move `JOB_FAMILITY_ID` as public constant to an internal class, e.g. `ICoreConstants`, and references it that here and int PDEPlugin.